test/csharp/csharp.nuget.props

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
please, remove this file from the PR.
In #2182 @douglaswth added it to `,gitignore`.

Particularly because this file contains env-specific line
```
<NuGetPackageRoot>C:\git\part4\Packages</NuGetPackageRoot>
```